lib/mix/tasks/phx.gen.auth/injector.ex

```diff
@@ -170,8 +170,8 @@ defmodule Mix.Tasks.Phx.Gen.Auth.Injector do
   def app_layout_menu_code_to_inject(%Schema{} = schema, padding \\ 4, newline \\ "\n") do
     already_injected_str = "#{schema.route_prefix}/log-in"
 
-    base_tailwind_classes = "text-[0.8125rem] leading-6 text-zinc-900"
-    link_tailwind_classes = "#{base_tailwind_classes} font-semibold hover:text-zinc-700"
+    base_tailwind_classes = "text-[0.8125rem] leading-6 text-base-content"
+    link_tailwind_classes = "#{base_tailwind_classes} font-semibold hover:text-base-content/80"
```

Comment:
I would love if those were simpler. Can we use `link` instead? Or a navbar? Or a menu in the `ul` list below?

Reply:
simplified